src/util/RedisBinaryDownload.ts

```diff
@@ -202,62 +219,87 @@ export default class RedisBinaryDownload {
       const fileStream = fs.createWriteStream(tempDownloadLocation);
 
       log(`trying to download https://${httpOptions.hostname}${httpOptions.path}`);
-      https
-        .get(httpOptions as any, (response) => {
-          // "as any" because otherwise the "agent" wouldnt match
-          if (response.statusCode != 200) {
-            if (response.statusCode === 404) {
-              reject(
-                new Error(
-                  'Status Code is 404\n' +
-                    "This means that the requested version doesn't exist\n" +
-                    `  Used Url: "https://${httpOptions.hostname}${httpOptions.path}"\n` +
-                    "Try to use different version 'new RedisMemoryServer({ binary: { version: 'X.Y.Z' } })'\n"
-                )
-              );
+
+      const get = () => {
+        https
+          .get(httpOptions as any, (response) => {
+            // "as any" because otherwise the "agent" wouldnt match
+            if (response.statusCode != 200) {
+              if (response.statusCode === 301 || response.statusCode === 302) {
```

Comment:
why did we need to add all this redirect logic? was it to support the redirect from `github.com` to `raw.githubusercontent.com`? it seems like we could just use the `raw.githubusercontent.com` directly, and then we shouldn't need to add all this code.

Reply:
yes, GitHub serves files from aws, it generates a temporary authorized URL first then redirects to it. It's good enough.

Reply:
can you give me an example of a URL that requires this code?

Reply:
i see it is different now, sometime ago github had this aws redirect. when I had a redirect i solved it.

now i see it gives file from 

https://raw.githubusercontent.com/ServiceStack/redis-windows/master/downloads/redis-latest.zip